src/sensai/featuregen.py

```diff
@@ -243,21 +243,30 @@ class FeatureGeneratorFlattenColumns(RuleBasedFeatureGenerator):
     Instances of this class take columns with vectors and creates a dataframe with columns containing entries of
     these vectors.
 
-    For example, if columns "vec1", "vec2" contain vectors of dimensions dim1, dim2, a datafrane dim1+dim2 new columns
+    For example, if columns "vec1", "vec2" contain vectors of dimensions dim1, dim2, a dataframe with dim1+dim2 new columns
     will be created. It will contain the columns "vec1_<i1>", "vec2_<i2>" with i1, i2 ranging in (0, dim1), (0, dim2).
 
     """
-    def __init__(self, columns: Union[str, Sequence[str]], categoricalFeatureNames: Sequence[str] = (),
+    def __init__(self, columns: Optional[Union[str, Sequence[str]]] = None, categoricalFeatureNames: Sequence[str] = (),
             normalisationRules: Sequence[data_transformation.DFTNormalisation.Rule] = (),
             generatedColumnsRuleTemplate: data_transformation.DFTNormalisation.RuleTemplate = None):
+        """
+
+        :param columns: column name, list of columns or None. If None, all columns in the input dataframe fill be flattened
```

Comment:
name of the column or list of names of columns to be flattened. If None, all columns will be flattened.
It's good to have the phrase "to be flattened" in there...

Reply:
Done